src/mgr/MgrPyModule.cc

```diff
@@ -171,30 +171,45 @@ int MgrPyModule::load()
   }
 
   // Find the class
-  // TODO: let them call it what they want instead of just 'Module'
-  auto pClass = PyObject_GetAttrString(pModule, (const char*)"Module");
-  Py_DECREF(pModule);
-  if (pClass == nullptr) {
+  auto  mgr_module_type = PyObject_GetAttrString(pModule, (const char*)"MgrModule");
+
+  if (mgr_module_type == nullptr) {
```

Comment:
i think the plugin is not required to expose `MgrModule`. can we load this class from `mgr_module` instead?